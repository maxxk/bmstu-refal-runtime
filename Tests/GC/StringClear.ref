$ENTRY Go
{
	= <Clear 'qwertyuiopasdfghjklzxcvbnm'>
}

Clear
{
	= Over;
	s.1 e.1	= <Clear e.1>;	
}