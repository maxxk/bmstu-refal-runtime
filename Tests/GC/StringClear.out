Heap size: 5120
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Start garbage collection.
Ok!
